src/ontology/ro-edit.owl

```diff
@@ -5828,6 +5829,16 @@ AnnotationAssertion(obo:IAO_0000115 obo:RO_0008508 "inverse of lays eggs on")
 AnnotationAssertion(rdfs:label obo:RO_0008508 "has eggs laid on by")
 SubObjectPropertyOf(obo:RO_0008508 obo:RO_0002619)
 
+# Object Property: obo:RO_0008509 (has roost)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0008509 "x 'has roost' y if and only if: x is an organism, y is a habitat, and y can support rest behaviors x.")
+AnnotationAssertion(dce:creator obo:RO_0008509 <http://orcid.org/0000-0002-2908-3327>)
+AnnotationAssertion(dce:date obo:RO_0008509 "2023-01-18T14:28:21Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:inSubset obo:RO_0008509 subsets:ro-eco)
+AnnotationAssertion(rdfs:comment obo:RO_0008509 "A population of xs will possess adaptations (either evolved naturally or via artifical selection) which permit it to rest in y.")
```

Comment:
can you please give a concrete example with terms in existing OBO ontologies as subject/predicate annotated with IAO:0000112 (example of usage)

Reply:
@cthoyt Do you mean just an example of how the term is used? E.g., "bird has roost some nest"

Reply:
yeah, that would be great, as long as it gives explicit references to Bird and Nest using OBO PURLs

Reply:
I don't think examples of usage have to use obo purls. Free text is fine.  The purpose of "example of usage" is to provide extra context to disambiguate the term.

Reply:
> This is for the benefit of future users, who without a concrete example might not be able to figure out how to use the relationship

I agree. That is the purpose of the annotation. We haven't required that OBO PURLs be used. See examples for `example of usage` in [adjacent to](http://purl.obolibrary.org/obo/RO_0002220), [capable of](http://purl.obolibrary.org/obo/RO_0002215), [characteristic of](http://purl.obolibrary.org/obo/RO_0000052), and many more ...

But @diatomsRcool if you can provide an example OBO PURLs, that would be nice.